prov/efa/src/rxr/rxr_cq.c

```diff
@@ -227,39 +227,93 @@ int rxr_cq_handle_tx_error(struct rxr_ep *ep, struct rxr_tx_entry *tx_entry,
 	return ofi_cq_write_error(util_cq, &err_entry);
 }
 
-/*
- * Queue a packet on the appropriate list when an RNR error is received.
+/* @brief Queue a packet that encountered RNR error and setup RNR backoff
+ *
+ * We uses an exponential backoff strategy to handle RNR errors.
+ *
+ * `Backoff` means if a peer encountered RNR, an endpoint will
```

Comment:
While maybe defining exponential is overkill, thank you for the detailed comments in this patch.